deploy/clowdapp.yml

```diff
@@ -154,6 +159,7 @@ objects:
       SEARCH_CLIENT_SECRET_STAGE: dGVzdFZhbHVl
       INTERCOM_DBAAS: dGVzdFZhbHVl
       INTERCOM_DBAAS_DEV: dGVzdFZhbHVl
+      DEBUG_FAVORITES_ACCOUNT_1: OTk5
```

Comment:
This is fine for now but it might be easier to handle multiple as an env var if needed in the future